bin/tx-prover/src/commands/mod.rs

```diff
@@ -0,0 +1,108 @@
+use clap::Parser;
+use init::Init;
+use proxy::StartProxy;
+use serde::{Deserialize, Serialize};
+use worker::StartWorker;
+
+pub mod init;
+pub mod proxy;
+pub mod worker;
+
+/// Configuration of the proxy.
+///
+/// It is stored in a TOML file, which will be created by the `init` command.
+/// It allows manual modification of the configuration file.
+#[derive(Serialize, Deserialize)]
+pub struct ProxyConfig {
+    /// List of workers to start
+    pub workers: Vec<WorkerConfig>,
```

Comment:
The comment here is a bit confusing: I don't think the proxy is starting these workers, right?